### DISCUSSION TITLE
Welcome to burst2safe Discussions!

### DISCUSSION THREAD

**Discussion by forrestfwilliams** [ASFHyP3/burst2safe#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dthuynguyen** [comment]
I am new about Sentient burst products
How to convert BURST to SAFE 
Example: 
S1_071093_IW2_20241015T002904_VV_8478-BURST.tiff
S1_071093_IW2_20241015T002904_VV_8478-BURST.xml

How to run burst2safe?
Thanks
